_infdemos/ingest-Large-Data.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Ingesting Externally Hosted Large Datasets
+main_author: Amy Paguirigan
+primary_reviewers: vortexing
+---
+
+Before ingesting any external data sets, please refer to the [Overview of Data Storage at Fred Hutch](/computing/store_overview/) page to begin to identify the final home of the data you are ingesting.  Choosing a location will depend on how large the data are, how often you need to access them, and what type of security concerns might apply to the dataset.  
+
+Once you have an idea where the data will need to live at Fred Hutch, you can begin the process of transferring them to Fred Hutch supported data storage systems.  
+
+Ideally, one wouldn't transfer large amounts of data intended to be temporary into systems like *Fast* storage as that data will be included in the frequent snapshots for backup purposes in that storage system.  This means these data will have a large footprint in the overall system, requiring long term backup of what are actually temporary data (ironic in a way). A better way to ingest and store large data sets is to first ingest the data into *Scratch* storage and then move them into secure, stable object storage systems like *Economy* (either *Local* or *Cloud*).  
+
+## FTP to *Scratch*
+First, make a directory in Scratch for the data you want to ingest.  Then use `wget` to copy, in this case recursively, the data from the ftp url (e.g., ftp.broadinstitute.org/bundle/) using the username (user) and password (pass) required.  
+
+```
+cd /fh/scratch/delete10/lastname_f/
+mkdir ingestedDataDir
+cd ingestedDataDir
+wget --recursive ftp://user@ftp.broadinstitute.org/bundle/
+```
+
+While `wget` supports sending a password in the command line (either by using `user:pass@` or via the `--password` option) this will make your credentials visible to anyone on the system.
+
+### Helpful `wget` Options
+
+#### Skipping Existing Files
+
+If you need to repeat the ftp if not all files come down, use no-clobber when resending the command and this will skip any files that already exist.
+
+```
+--no-clobber
+```
+
+#### Managing Credentials
```

Comment:
So I added this big blob of text on managing credentials for wget and then realized that info should likely live in a document devoted to useful wget chicanery.  Can it live here for now until we find a good home for that?

Reply:
Let's let it grow here until it needs a new home!  wget and it's gory details are welcome here!  It's searchable here, and I think this can grow a bit regarding ingesting large data sets, and then perhaps the detailed linux/code chunks when they get large enough can be linked compdemos.  Especially if that content would ideally be linked to from another page in the Wiki.